conf/application.conf

```diff
@@ -75,17 +91,23 @@ application.router=prod.Routes
 
 controllers {
```

Comment:
Can probably be removed, also the 'evolution', play modules and session timeout sections are just noise